include/nlohmann/json.hpp

```diff
@@ -2193,6 +2261,9 @@ class basic_json
         using std::swap;
         swap(m_type, other.m_type);
         swap(m_value, other.m_value);
+#if JSON_DIAGNOSTICS
+        m_parent = other.m_parent;
```

Comment:
Here is essentially the same issue as with the move constructor above.

Reply:
Agreed.